server/zanata-frontend/src/frontend/app/styles/style.less

```diff
@@ -6035,6 +6035,14 @@ i.i--left.i--lock, i.i--left.i--trash {
   padding-bottom:@spacing-base-and-a-half;
 }
 
+.translationContainer pre.cm-s-default {
+  word-break: break-word;
```

Comment:
break-all does not work. break-word is a webpack specific value. Does break-all work for you?